collectorApp.php

```diff
@@ -1,5 +1,19 @@
 <?php
 
+$connectionString = 'mysql:host=db; dbname=planetdatabase';
+$dbUsername = 'root';
+$dbPassword = 'password';
+$db = new PDO($connectionString, $dbUsername, $dbPassword);
+
+$queryString = 'SELECT * FROM `myplanets`;';
+$query = $db->prepare($queryString);
+$query->execute();
+
+$allResults = $query->fetchAll();
+$firstResult = $query->fetch();
```

Comment:
do you need both an $allResults variable and $firstResult

Reply:
Thanks :)

Reply:
its more to see if it works, will delete before main